### ISSUE TITLE
-p broken

### ISSUE THREAD

**Issue by lacombar** [bogado/file-line#53]
When using the following command:

```
% vim -p foo:10 bar:20
```

Only `foo` is opened.

**Comment by lacombar** [comment]
Might be related to this: https://code.google.com/p/vim/issues/detail?id=293

**Comment by bogado** [comment]
Yes that's definitely related to that. :+1:  nice finding, too bad it don't have a solution.

**Comment by lacombar** [comment]
FYI, I rewrote most of the script to fit my need, cf https://github.com/lacombar/vim-file-line/commit/63d6914316ac48217abdee924c7db553f4380d76

**Comment by bogado** [comment]
I was looking in the documentation and the strategy of rewriting the arguments is not a good one.
